worlds/subnautica/Locations.py

```diff
@@ -571,9 +571,35 @@ class LocationDict(TypedDict, total=False):
             'need_laser_cutter': False,
             'position': {'x': 83.2, 'y': -276.4, 'z': -345.5}},
 }
+
 if False:  # turn to True to export for Subnautica mod
```

Comment:
you could possibly have that in a separate file you execute using `python -m worlds.subnautica.export` or something - maybe as a future idea.